tools/debug-ui/src/ChainInfoSummaryView.tsx

```diff
@@ -0,0 +1,96 @@
+import './ChainInfoSummaryView.scss';
+import { useMemo } from 'react';
+import { useQuery } from 'react-query';
+import { fetchChainProcessingStatus, fetchFullStatus } from './api';
+
+type ChainInfoSummaryViewProps = {
+    addr: string;
+};
+
+export const ChainInfoSummaryView = ({ addr }: ChainInfoSummaryViewProps) => {
+    const {
+        data: fullStatus,
+        error: fullStatusError,
+        isLoading: fullStatusLoading,
+    } = useQuery(['fullStatus', addr], () => fetchFullStatus(addr));
+    const {
+        data: chainProcessingInfo,
+        error: chainProcessingInfoError,
+        isLoading: chainProcessingInfoLoading,
+    } = useQuery(['chainProcessingInfo', addr], () => fetchChainProcessingStatus(addr));
+    const {
+        chainInfoHead,
+        chainInfoHeaderHead,
+        numBlocksOrphanPool,
+        numBlocksMissingChunksPool,
+        numBlocksProcessing,
+    } = useMemo(() => {
+        let chainInfoHead = '';
+        let chainInfoHeaderHead = '';
+        let numBlocksOrphanPool = -1;
+        let numBlocksMissingChunksPool = -1;
+        let numBlocksProcessing = -1;
+        if (fullStatus && chainProcessingInfo) {
+            const head = fullStatus.detailed_debug_status!.current_head_status;
+            chainInfoHead += head.hash;
+            chainInfoHead += '@';
+            chainInfoHead += head.height;
+            const headerHead = fullStatus.detailed_debug_status!.current_header_head_status;
+            chainInfoHeaderHead += headerHead.hash;
+            chainInfoHeaderHead += '@';
+            chainInfoHeaderHead += headerHead.height;
+            const chainInfo = chainProcessingInfo.status_response.ChainProcessingStatus;
+            numBlocksOrphanPool = chainInfo.num_orphans;
+            numBlocksMissingChunksPool = chainInfo.num_blocks_missing_chunks;
+            numBlocksProcessing = chainInfo.num_blocks_in_processing;
+        }
+        return {
+            chainInfoHead,
+            chainInfoHeaderHead,
+            numBlocksOrphanPool,
+            numBlocksMissingChunksPool,
+            numBlocksProcessing,
+        };
+    }, [fullStatus, chainProcessingInfo]);
+    if (fullStatusLoading || chainProcessingInfoLoading) {
+        return <div>Loading...</div>;
+    } else if (fullStatusError || chainProcessingInfoError) {
+        return (
+            <div className="chain-and-chunk-info-view">
+                <div className="error">
+                    {((fullStatusError || chainProcessingInfoError) as Error).stack}
+                </div>
+            </div>
+        );
+    } else if (!fullStatus || !chainProcessingInfo) {
+        return (
+            <div className="chain-and-chunk-info-view">
+                <div className="error">No Data</div>
+            </div>
+        );
+    }
+    return (
+        <div className="chain-info-summary-view">
+            <p>
+                {' '}
```

Comment:
Do you need the spaces before and after? It's already wrapped in a `<p>` so I don't feel like it would do anything.